src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -91,6 +91,25 @@ <h4 *ngIf="course.id" id="jhi-course-heading-edit" jhiTranslate="artemisApp.cour
                         <jhi-date-time-picker labelName="{{ 'artemisApp.course.endDate' | translate }}" formControlName="endDate"></jhi-date-time-picker>
                     </div>
                 </div>
+                <div class="form-check">
+                    <input
+                        type="checkbox"
+                        class="form-check-input"
+                        name="testCourse"
+                        id="field_testCourse"
+                        formControlName="testCourse"
+                        [checked]="course.testCourse"
+                        (change)="changeTestCourseEnabled()"
+                    />
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.testCourse.title" for="field_testCourse"></label>
+                    <fa-icon [icon]="'question-circle'" class="text-secondary" placement="top" ngbTooltip="{{ 'artemisApp.course.testCourse.description' | translate }}"></fa-icon>
+                </div>
+                <div class="form-group" *ngIf="!course.testCourse">
```

Comment:
this should not be mutually exclusive. We should be able to set `test` and `semester` and then we should also display both